payment/controllers.go

```diff
@@ -56,6 +56,7 @@ func Router(service *Service) chi.Router {
 
 	r.Method("OPTIONS", "/{orderID}", middleware.InstrumentHandler("GetOrderOptions", corsMiddleware([]string{"GET"})(nil)))
 	r.Method("GET", "/{orderID}", middleware.InstrumentHandler("GetOrder", corsMiddleware([]string{"GET"})(GetOrder(service))))
+	r.Method("DELETE", "/{orderID}", middleware.InstrumentHandler("CancelOrder", corsMiddleware([]string{"DELETE"})(CancelOrder(service))))
```

Comment:
I don't think we should have cancel order without authentication

Reply:
Simple token should do right?  This is initiated from subscriptions?

Reply:
added authentication: https://github.com/brave-intl/bat-go/pull/889/commits/df62ffdda5dfffd1eea14232563c78228af9d6c9